test/testdata/namer/fuzz_type_template_overwrite.rb.symbol-table-raw.exp

```diff
@@ -3,7 +3,7 @@ class <C <U <root>>> < <C <U Object>> ()
     method <S <C <U <root>>> $1>#<N <U <static-init>> $CENSORED> (<blk>) @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=3:1 end=6:4}
       argument <blk><block> @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=??? end=???}
   class <C <U A>> < <C <U Object>> () @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=3:1 end=3:8}
-    static-field <C <U A>>::<C <U B>> -> AliasType { symbol = <S <C <U A>> $1>::<C <U B>> } @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=5:3 end=5:20}
+    static-field <C <U A>>::<C <U B>> -> Integer @ Loc {file=test/testdata/namer/fuzz_type_template_overwrite.rb start=5:3 end=5:20}
```

Comment:
I guess this is also due to the change to resolver?

Reply:
Yes.